lib/diagnostic.js

```diff
@@ -22,7 +32,15 @@ const uniqueWords = function (/* */) {
 // a second argument.
 
 const wordCount = function (/* */) {
-  /* your response here */
+
+  function WordCount(str) {
+  let totalSoFar = 0;
+  for (let i = 0; i < WordCount.length; i++)
```

Comment:
You don't want to be checking `WordCount.length` in here, as `WordCount` is simply the name of your function